packages/loot-core/src/server/budget/goal-template.pegjs

```diff
@@ -43,7 +43,7 @@ upTo = 'up'i _ 'to'i
 _ 'space' = ' '+
 d 'digit' = [0-9]
 number 'number' = $(d+)
-amount 'amount' = currencySymbol? _? amount: $(d+ ('.' d d)?) { return +amount }
+amount 'amount' = currencySymbol? _? amount: $(d+ ('.' d+ (d)?)?) { return +amount }
```

Comment:
```suggestion
amount 'amount' = currencySymbol? _? amount: $(d+ ('.' (d d?)?)?) { return +amount }
```

That’s a bit confusing but it should allow `123`, `123.`, `123.4`, and `123.45`.

Reply:
Indeed it is.  I didn't think to test 123. as a valid input.